Jenkinsfile

```diff
@@ -45,7 +45,9 @@ ansiColor('xterm') {
 
             sshagent(['d8533977-c4c5-4e2b-938d-ae7fcbe27aac']) {
               // return the exit code because we don't care about failures
-              sh script: 'git remote add upstream git@github.com:ciscospark/react-ciscospark.git', returnStatus: true
+              sh script: 'git remote add upstream git@github.com:webex/react-ciscospark.git', returnStatus: true
+              // renaming just in case it already exists
+              sh script: 'git remote set-url upstream git@github.com:webex/react-ciscospark.git', returnStatus: true
```

Comment:
Decided not to go with the `webex` remote name as to not appear like it was different than what everyone thinks of as `upstream`.